src/GovUk.Education.ExploreEducationStatistics.Content.Model/Services/PublicReleaseFileBlobService.cs

```diff
@@ -16,36 +16,36 @@ namespace GovUk.Education.ExploreEducationStatistics.Content.Model.Services;
 
 public class PublicReleaseFileBlobService : IReleaseFileBlobService
```

Comment:
Feels like `PublicReleaseFileBlobService` and `PrivateReleaseFileBlobService` would be nicer handled in the same way as you've done with BlobStorageService, IPublicStorageService and IPrivateStorageService, but one for another PR.  Maybe we should think about breaking out a ticket for this?

Reply:
Created EES-4344

Reply:
Cool!